src/emailservice/Dockerfile

```diff
@@ -42,4 +39,4 @@ RUN apk add --no-cache libstdc++
 COPY . .
 
 EXPOSE 8080
-ENTRYPOINT [ "python", "email_server.py" ]
\ No newline at end of file
+ENTRYPOINT [ "python", "-u", "email_server.py" ]
```

Comment:
I recommend keeping this PYTHONUNBUFFERED=0 so that all processes (like `pip`) benefit from it.

Reply:
Done.